app/views/users/show.html.erb

```diff
@@ -8,14 +8,12 @@
       <!-- If the user doesn't have any tokens, prompt them to create one. -->
       <% if @user.tokens.empty? %>
         <%= render 'tokenprompt' %>
-      
+
       <!-- If all of the user's tokens are expired, let them know that and prompt them to create a new one -->
       <% elsif @user.tokens_expired? %>
         <%= render 'tokenprompt', token_prompt: "Your ORCID token has expired. Please click the button below to re-authorize Princeton University to access your ORCID record." %>
 
-        <!-- If the user has a valid token, show them their ORCID iD and tokens with expiration dates -->
-      <p>ORCID iD, @user.valid_token, is connected to your NetID, @user.uid</p>
-      <p>Your token grants Princeton access to read and update your ORCID record until @user.valid_token.expiration.strftime('%m/%d/%Y').</p>
```

Comment:
If all of a user's tokens are expired, then @user.valid_token will always return nil. I assume this was originally a part of the valid token section of the display but did not get removed after the `_user` partial was made.  I moved `<p>ORCID iD, @user.valid_token, is connected to your NetID, @user.uid</p>` over to the `_user` partial since it makes more sense for the ORCID iD connection to the NetID to be displayed there